components/index.js

```diff
@@ -13,15 +24,24 @@ export {
   IconButton,
   Dialog,
   FontIcon,
+  Heading1,
+  Heading2,
+  Heading3,
+  Heading4,
   Menu,
+  Monospaced,
   IconMenu,
   MenuItem,
   MenuDivider,
   LoadingMolecule,
+  OldStyle,
```

Comment:
Not too fond of `OldStyle` as name, it's not obvious enough. `OldStyleNumber`? Something less vague at least 👍